workflow_tests/utils/preGenerateTest.ts

```diff
@@ -268,13 +279,13 @@ console.log(`Creating mock file ${mockFilePath}`);
 fs.writeFileSync(mockFilePath, mockFileContent);
 console.log(`Mock file ${mockFilePath} created`);
 
-const assertionsFileContent = getAssertionsFileContent(workflowName, workflowJobs);
+const assertionsFileContent = getAssertionsFileContent(workflowJobs);
```

Comment:
Though this updates makes sense, make sure it doesn't have side effects

Reply:
it's the only place this function is being used and this param wasn't doing anything inside of the function